File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/shelters/CreateShelterCommand.java
package by.fpmibsu.findafriend.controller.commands.shelters;

import by.fpmibsu.findafriend.application.mediatr.Request;

public class CreateShelterCommand extends Request<Boolean> {
    public String name;
    public String address;
    public String website;
    public int placeId;
}

File: BackendNET/Backend/Commands/Shelters/CreateShelter/CreateShelterCommand.cs
using MediatR;

namespace Backend.Commands.Shelters.CreateShelter;

public class CreateShelterCommand : IRequest<bool>
{
    public string Name { get; set; } = "";
    public string Address { get; set; } = "";
    public string Website { get; set; } = "";
    public int PlaceId { get; set; }
}